src/containers/signupForm/index.tsx

```diff
@@ -0,0 +1,509 @@
+                },
+              ]}
+              style={{ display: 'inline-block' }}
+            >
+              <Checkbox>
+                No family member or attendee will visit The Sajni Center if they
+                have been sick in the past 24 hours, are feeling ill, have been
+                exposed to a virus, or do not have the appropriate age required
+                immunizations.
+              </Checkbox>
+            </Form.Item>
+
+            <Form.Item
+              name="check2"
+              valuePropName="checked"
+              rules={[
+                {
+                  required: true,
+                  type: 'enum',
+                  enum: [true],
+                  message: 'Please check all boxes',
+                },
+              ]}
+              style={{ display: 'inline-block' }}
+            >
+              <Checkbox>
+                All parents are to remain at The Sajni Center during programs
+              </Checkbox>
+            </Form.Item>
+
+            <Form.Item
+              name="check3"
+              valuePropName="checked"
+              rules={[
+                {
+                  required: true,
+                  type: 'enum',
+                  enum: [true],
+                  message: 'Please check all boxes',
+                },
+              ]}
+              style={{ display: 'inline-block' }}
+            >
+              <Checkbox>
+                My children are up to date on all vaccinations and I will
+                provide a copy of my children’s immunization records prior to
+                attending any programs. (Children who are being treated for
+                cancer often have severely compromised immune systems, so we are
+                required to collect this information for their safety. All
+                medical information will be stored in a HIPPA-compliant manner.)
+              </Checkbox>
+            </Form.Item>
+            <Form.Item
+              label="Head of Family Name"
+              name="headOfFamilyName"
+              className="stacked-input inline-block-half"
+              rules={[
+                {
+                  required: true,
+                  message: 'Please input the name of the head of the family',
+                },
+              ]}
+            >
+              <Input placeholder="Head of Family Name" />
+            </Form.Item>
+
+            <Form.Item
+              label="Head of Family Initials"
+              name="headOfFamilyInitials"
+              className="stacked-input inline-block-half"
+              rules={[
+                {
+                  required: true,
+                  message: "Please input head of family's initials",
+                },
+              ]}
+            >
+              <Input placeholder="Head of Family Initials" />
+            </Form.Item>
+
+            <Form.Item
+              label="Date of Signature"
```

Comment:
Signature? Huh?

Reply:
We're moving that "confirmation" stuff into this signup form because we got rid of the confirmation component entirely.